docs/content/post/developer-documentation.md

```diff
@@ -0,0 +1,310 @@
++++
+author = "Daniel Mellado"
+title = "Developer Documentation"
+date = "2020-10-12"
+description = "Developer Docs for Egress Router CNI"
+tags = [
+    "cni",
+    "egress",
+    "developer"
+]
++++
+
+This article covers how to install and test the Egress Router CNI, both using
+cnitool or deploying openshift and using its official image.
+
+## Using cnitool
+
+[cnitool](https://github.com/containernetworking/cni/tree/master/cnitool) is
+a simple program that executes a CNI configuration. It will add or
+remove an interface in an already-created network namespace. You can use it in
```

Comment:
nit: Should we be using first, second or third person in this documentation? I'm not sure on the protocol here @jboxman, but it my mind wouldn't we be avoiding personal pronouns?

Reply:
@astoycos, it depends. I think outside of OpenShift docs, the style is undefined. For OCP docs, we do use _you_.